via/CMakeLists.txt

```diff
@@ -66,31 +66,14 @@ elseif (MSVC)
     target_compile_definitions(vkvia PRIVATE _CRT_SECURE_NO_WARNINGS _USE_MATH_DEFINES)
 endif()
 
-# NOTE: jsoncpp hasn't had updates in a bit and isn't getting fixes.
-# may be worth using nlohmann/json in the future which gets more updates
-# and has better CMake support.
-find_package(jsoncpp CONFIG)
-if (TARGET jsoncpp_static)
-    target_link_libraries(vkvia PRIVATE jsoncpp_static)
-    
-# Support using jsoncpp.pc but only for UNIX platforms.
-# And only if UPDATE_DEPS is disabled.
-elseif (UNIX AND NOT UPDATE_DEPS)
-
-    # https://github.com/LunarG/VulkanTools/issues/1908
-    find_package(PkgConfig)
-    if (PKG_CONFIG_FOUND)
-        pkg_check_modules(jsoncpp REQUIRED IMPORTED_TARGET jsoncpp)
-        target_link_libraries(vkvia PRIVATE PkgConfig::jsoncpp)
-    endif()
-
-endif()
+find_package(jsoncpp 1.9.6 CONFIG)
```

Comment:
Even if the build uses meson it will still package a CMake configuration file. So now the CMake code can be cleaned up 💯

Reply:
While this code could potentially be cleaned up. I have reverted it since package managers could still use an older version of jsoncpp and this could cause an issue.

I'm rebased the PR to the bare minimum change needed to fix CMake 4.0 support